docs/src/modules/ROOT/pages/optimization-algorithms/overview.adoc

```diff
@@ -637,6 +637,126 @@ This is useful for benchmarking.
 
 Switching xref:using-timefold-solver/running-the-solver.adoc#environmentMode[EnvironmentMode] can heavily impact when this termination ends.
 
+[#diminishedReturnsTermination]
+==== `DiminishedReturnsTermination`
+
+Terminates when the rate of improvement is below a percentage of the initial rate of improvement.
+The rate of improvement is the score difference between the final best score and the initial best score in a window, divided by the duration of the window.
+The first window is called a grace period, which cannot trigger a termination.
+
+image::optimization-algorithms/overview/diminishedReturnsTermination.png[align="center"]
+
+[NOTE]
+====
+If the hard score improves, the grace period is reset and the initial rate of improvement is recalculated.
+====
```

Comment:
Since this is a core part of the algorithm, I don't think it should be a NOTE. Rather, a normal text like everything else.

Also, the above comment about the softest score will pay off here, where we talk about the hard score all of a sudden.